### ISSUE TITLE
Function of the power button

### ISSUE THREAD

**Issue by mathebox** [B5r1oJ0A9G/teufel_raumfeld#72]
I noticed that the function of the power button is a bit inconsistent. While grouped devices are ungrouped, pressing the button for a single speaker just pauses the playback.

#### Grouped devices
- Media playback on two grouped speakers: speaker1 (host) and speaker2
- Press the power button on the media card for the device group
- Playback is paused 
- Media Card for the device group is deactivated
- Media card for the host speaker becomes active
- Both devices remain in power state `on`

#### Single speakers
- Media playback on a single speaker 
- Press the power button on the respective media card
- Playback is paused 
- Media card for the speaker remains active  
- The device remains in power state on

### Inconsistent and somewhat confusing behavior 
- For single speakers, the functionality of the power button and the play/pause button does not differ.
- The power state is not affected by the power button. (Using the power state select for controlling the power state in automations works perfectly fine.)
----------------
Do you have any suggestions for making the behavior more consistent?

**Comment by B5r1oJ0A9G** [comment]
Indeed, if I remember correctly, when developing it the power button I intended to use for group and un-grouping only. Actually even only for grouping, to activate the corresponding speaker or speaker group. This worked for me, since I do have the spakers always on or in automatic stand-by.

What implemention would you propose instead?
